suites/pacific/cephfs/tier_1_cephfs_mirror.yaml

```diff
@@ -0,0 +1,141 @@
+#===============================================================================================
+# Tier-level: 1
+# Test-Suite: tier_1_cephfs_mirror.yaml
+# Test-Case: Configure CephFS Mirror setup and run IOs
+#
+# Cluster Configuration:
+#    No of Clusters : 2
+#    Each cluster configuration
+#    6-Node cluster(RHEL-8.3 and above)
+#    3 MONS, 2 MGR, 2 MDS, 3 OSD and 1 CEPHFS MIRROR, 1 Client service daemon(s)
+#     Node1 - Mon, Mgr, Installer
+#     Node2 - Mon, Mgr
+#     Node3 - Mon, OSD
+#     Node4 - OSD,MDS
+#     Node5 - OSD, MDS
+#     Node6 - CephFS Mirror
+#     Node7 - Client
```

Comment:
Cluster 2 has no node with id `7`

Reply:
We don't need cephfs-mirror node on cluster2 ,

May be I can mention as "#     Node6 - CephFS Mirror (Only on 1st cluster)"

Reply:
IMO, Splitting up the cluster conf would be good in the comments.

Reply:
Done.